docs/src/main/tut/parsing.md

```diff
@@ -60,4 +60,16 @@ import cats.syntax.either._
 val json: Json = parse(rawJson).getOrElse(Json.Null)
 ```
 
+## Warnings and known issues
+
+For Scala.js, the result of parsing JSON is restricted to the expressiveness of
+the JavaScript types. In particular numerical values may [loose precision][#393]
+when decoded by [`JSON.parse`][json.parse] to a JavaScript number. For example:
```

Comment:
I'm not sure "JavaScript number" is exactly the right phrasing here—it seems more like the problem is that you're trying to decode to a Scala type like `Long` that has its own semantics, even if it's implemented using JavaScript numbers on Scala.js. I'm happy to work on details like this in a follow-up PR, though.